code/modules/mob/camera/imaginary_friend.dm

```diff
@@ -82,6 +88,51 @@
 
 	return ..()
 
+/mob/camera/imaginary_friend/verb/toggle_darkness()
+	set category = "Imaginary Friend"
+	set name = "Toggle Darkness"
+
+	switch(lighting_alpha)
+		if(LIGHTING_PLANE_ALPHA_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+		if(LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+		if(LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_INVISIBLE
+		else
+			lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
+
+	update_sight()
+
+/mob/camera/imaginary_friend/verb/update_outfit()
+	set category = "Imaginary Friend"
+	set name = "Change Appearance"
+
+	var/outfit_choice = tgui_input_list(usr, "Choose your appearance:", "[src]", outfit_choices)
```

Comment:
This needs a if(!output_choice) return

Reply:
Good catch. If you were to cancel currently you become a naked human.